shell/browser/native_window_views.h

```diff
@@ -174,7 +175,26 @@ class NativeWindowViews : public NativeWindow,
   TaskbarHost& taskbar_host() { return taskbar_host_; }
 #endif
 
+#if defined(OS_WIN)
+  enum class TitleBarStyle{
+      kNormal,
+      kHidden,
+  };
+  TitleBarStyle title_bar_style() const { return title_bar_style_; }
+  bool IsWindowControlsOverlayEnabled() const {
+    return (title_bar_style_ == NativeWindowViews::TitleBarStyle::kHidden) &&
+           titlebar_overlay_;
+  }
+  SkColor overlay_color() const { return overlay_color_; }
+  SkColor overlay_symbol_color() const { return overlay_symbol_color_; }
+#endif
+
  private:
+#if defined(OS_WIN)
+  TitleBarStyle title_bar_style_ = TitleBarStyle::kNormal;
+  bool titlebar_overlay_ = false;
```

Comment:
The `titlebar_overlay_` has already been declared and parsed in the base `NativeWindow` class.

Also I think we can move `title_bar_style_` to base `NativeWindow` class too.

Reply:
Ok that makes sense. I went ahead and moved it in the commit "move title_bar_style_ to NativeWindow".